roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -5,7 +5,7 @@ apiLevels:
 - v1
 apiVersion: v1
 assetConfig:
-  logoutURL: ""
+  logoutURL: {{ openshift.master.logout_url }}
```

Comment:
It would be if we didn't throw away any values that are set to "" or None in the openshift_facts module, which gives us the ability to allow overriding the defaults and not persisting None or "" to the local facts file we generate, only the values that we actually intend to override. It also makes merging the persisted facts with the provided facts and the new set of persisted facts with the default facts a bit easier.